contracts/allocs/scripts/genallocs.go

```diff
@@ -32,12 +32,21 @@ func main() {
 	fmt.Println("genallocs succeeded")
 }
 
+// After a persistent network has been deployed, lock it's allocs.
+var locked = map[netconf.ID]bool{
```

Comment:
Should we move this map to the `allocs` package so it lives with the actual persistent allocs?

Reply:
And export a func like this 

`func IsLocked(id netconf.ID) bool `

Reply:
yeah I like that